common/src/main/java/com/redhat/cloud/notifications/routers/endpoints/EndpointTestRequest.java

```diff
@@ -1,23 +1,27 @@
 package com.redhat.cloud.notifications.routers.endpoints;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
 
-import javax.validation.constraints.NotBlank;
-import javax.validation.constraints.NotNull;
-import java.util.UUID;
-
+/**
+ * Represents the structure of the "test endpoint" request that the user might
+ * send.
+ */
 @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
-public class EndpointTestRequest {
-
-    @NotNull
-    public UUID endpointUuid;
+public final class EndpointTestRequest {
+    public String message;
 
-    @NotBlank
-    public String orgId;
+    public EndpointTestRequest(@JsonProperty("message") final String message) {
```

Comment:
Is that `@JsonProperty` annotation needed?

Reply:
I think so, because if I remove it, then I start getting the following errors:

```com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `com.redhat.cloud.notifications.routers.endpoints.EndpointTestRequest` (although at least one Creator exists): cannot deserialize from Object value (no delegate- or property-based Creator)
```

Reply:
Apparently I can [add an empty constructor as well](https://stackoverflow.com/a/53192674/6319771), as an alternative. But I think it is just cleaner to have a single constructor, even if it is annotated... What do you think, though?